build-test.xml

```diff
@@ -30,6 +30,7 @@
         </fileset>
         <fileset dir="${java.home}/../lib">
             <include name="**/*.jar" />
+            <exclude name="missioncontrol/**" />
```

Comment:
I'm curious about the nature of those Oracle 8 JDK eclipse libs
When you're in the office next time, I'd like learn some of these things over lunch... to come up to speed ... just in case I need to lend a hand in anything in future